### ISSUE TITLE
re-enable oxli script

### ISSUE THREAD

**Issue by mr-c** [dib-lab/khmer#1275]
after the 2.0 release is out

**Comment by standage** [comment]
Punting this for now.

**Comment by betatim** [comment]
Could we add a description what exactly this means?

**Comment by mr-c** [comment]
`oxli normalize-by-median [...]` as the main method of running Oxli tools

https://github.com/dib-lab/khmer/blob/master/oxli/__init__.py

**Comment by betatim** [comment]
So the plan would be to move all of `scripts/*py` to use this new way of running them?

**Comment by mr-c** [comment]
That was the plan, as the scripts have very generic names. In the Debian package we implemented this as `khmer [...]` with a note to users on how to enable the legacy script names.

**Comment by standage** [comment]
Some of the supporting architecture is already in place. See `scripts/load-graph.py` and `oxli/build_graph.py`.

**Comment by mr-c** [comment]
At [one time the plan](http://khmer.readthedocs.io/en/v2.0/roadmap.html#remainder-of-v2-x-series) was to refactor in place adding `oxli` commands iteratively and switch to just `oxli [...]` for 3.0

**Comment by standage** [comment]
Yes, that is still the plan...in principle, at least. Not much effort has yet been expended in that direction.

Once the main method of each script has been captured in reusable functions, maintenance of multiple entry points should be pretty trivial.
